### ISSUE TITLE
ComboBox: with fails to scroll selected item into view

### ISSUE THREAD

**Issue by majornista** [adobe/react-spectrum#4071]
# 🐛 Bug Report 

This bug is a duplicate of #3944.

When an item has been selected within a ComboBox with a large number of suggestions, the selected item does not scroll into view when the listbox gets expanded using ArrowDown or by clicking the Show Suggestions button.

## 🤔 Expected Behavior

When the listbox opens, the selected item should have focus and scroll into view within the listbox.

## 😯 Current Behavior

The selected item does not scroll into view. On desktop, ArrowUp or ArrowDown seems to focus the previous or next item from the selected item, so perhaps the focusedKey is getting set. But on mobile, the selected item could be far down in the list, where it would take many swipes to reach it using explore by touch.

https://user-images.githubusercontent.com/154077/219210409-19bbf080-4840-46dd-865d-e10e64d5d563.mp4

## 🔦 Context

Accessibility Audit

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| react-spectrum   |  [@adobe/react-spectrum@3.24.1](https://github.com/adobe/react-spectrum/releases/tag/%40adobe%2Freact-spectrum%403.24.1)
| Browser          | Chrome Version 110.0.5481.77 (Official Build) (arm64)
| Operating System | macOS Ventura 13.2.1 (22D68) / iOS / Android

## 🧢 Your Company/Team

Adobe/Accessibility

**Comment by LFDanLu** [comment]
Closing this out in favor of #3944 as you noted. I've added that issue to our board with the appropriate meta data
